ReportStats.py

```diff
@@ -0,0 +1,127 @@
+"""
+ReportStats.py
+--------------
+    This module provides a class, `ReportStats`, that holds the calculation results for the reports.
+
+Usage
+-----
+    from ReportStats import ReportStats
+
+    # Create an instance of ReportStats
+    report_stats = ReportStats()
+
+    # Set the extreme values and their corresponding dates
+    report_stats.set_extremes(highest, day_of_highest, lowest, day_of_lowest, humidity, most_humid_day)
+
+    # Set the average values
+    report_stats.set_averages(highest_avg_temperature, lowest_avg_temperature, avg_mean_humidity)
+"""
+
+
+class ReportStats:
+    """A class that will hold the calculations results.
+
+    Attributes
+    ----------
+    average_of_mean_humidity : int >= 0
+        Holds the average of the mean humidity of the month.
+
+    lowest_average_temperature : int >= 0
+        Holds the minimum of the average temperatures from the readings of the month.
+
+    highest_average_temperature : int >= 0
+        Holds the maximum of the average temperatures from the readings of the month.
+
+    humidity : int >= 0
+        Holds the value of the maximum humidity of the month.
+
+    lowest_temperature_day : str
+        Date when the lowest temperature was recorded in string format.
+
+    lowest_temperature : int >= 0
+        Holds the minimum temperature value of the month.
+
+    highest_temperature_day : str
+        Date when the highest temperature was recorded in string format.
+
+    highest_temperature : int >= 0
+         Holds the maximum temperature value of the month.
+
+    most_humid_day : str
+        Date when the most humidity was recorded in string format.
+
+    Methods
+    -------
+    set_extremes
+        Assign the extreme(highest, lowest) values of temperature ad humidity ad record the date when it occurred.
```

Comment:
Still not addressed yet. Please look at your code to find all the typos.

Reply:
There's an issue with the N key. I have resolved it now.